app/src/main/java/org/mifos/mobilebanking/ui/fragments/ThirdPartyTransferFragment.java

```diff
@@ -190,8 +191,8 @@ public void showUserInterface() {
         spPayFrom.setAdapter(payFromAdapter);
         spPayFrom.setOnItemSelectedListener(this);
 
-        beneficiaryAdapter = new ArrayAdapter<>(getActivity(), android.R.layout.simple_spinner_item,
-                listBeneficiary);
+        beneficiaryAdapter = new BeneficiarySpinnerAdapter(getActivity(),
+                R.layout.beneficiary_spinner_layout, listBeneficiary);
```

Comment:
Spinner Title should be `Select Beneficiary`

Reply:
@therajanmaurya I have made this change and update the image for this PR.